python/tank/util/metrics.py

```diff
@@ -204,25 +212,40 @@ def workers(self):
         """A list of workers threads dispatching metrics from the queue."""
         return self._workers
 
+
 class MetricsDispatchWorkerThread(Thread):
-    """Worker thread for dispatching metrics to sg logging endpoint.
+    """
+    Worker thread for dispatching metrics to sg logging endpoint.
 
     Once started this worker will dispatch logged metrics to the shotgun api
-    endpoint. The worker retrieves any pending metrics after the
+    endpoint, if available. The worker retrieves any pending metrics after the
     `DISPATCH_INTERVAL` and sends them all in a single request to sg.
 
-    In the case metrics dispatch isn't supported by the shotgun server,
-    the worker thread will exit early.
+    This worker will also fire the `log_metrics` hooks.
     """
 
-    API_ENDPOINT = "api3/log_metrics/"
-    """SG API endpoint for logging metrics."""
+    API_ENDPOINT = "api3/track_metrics/"
 
     DISPATCH_INTERVAL = 5
     """Worker will wait this long between metrics dispatch attempts."""
 
+    DISPATCH_SHORT_INTERVAL = 0.1
+    """
+    Delay in seconds between the posting of consecutive batches within a 
+    dispatcher cycle.
+    """
+
     DISPATCH_BATCH_SIZE = 10
-    """Worker will dispatch this many metrics at a time, or all if <= 0."""
+    """
+    Worker will dispatch this many metrics at a time, or all if <= 0.
+    NOTE: that current SG server code reject batches larger than 10.
+    """
+
+    # List of Event names suported by our backend
+    SUPPORTED_EVENTS = [
```

Comment:
minor: suggest formatting these as one event string per line. that'll make it clearer in the history when we add/remove individual events.